servers/xr/xr_interface.h

```diff
@@ -150,6 +153,14 @@ class XRInterface : public RefCounted {
 	virtual XRInterface::EnvironmentBlendMode get_environment_blend_mode() const { return XR_ENV_BLEND_MODE_OPAQUE; }
 	virtual bool set_environment_blend_mode(EnvironmentBlendMode mode) { return false; }
 
+	/* VRS */
+	float get_vrs_min_radius() const;
+	void set_vrs_min_radius(float p_vrs_min_radius);
+	float get_vrs_strength() const;
+	void set_vrs_strength(float p_vrs_strength);
```

Comment:
I don't really like putting all this on `XRInterface`. I feel like everything on `XRInterface` should be virtual and overridden by the specific interfaces (ie. `OpenXRInterface`, `WebXRInterface`, etc). If we just need to store some data, perhaps this should be on `XRServer`?

Reply:
The reason I put it here because we now have a fallback that works in all cases. All interfaces can piggy back on this. Some XR runtimes may have their own implementation but interestingly, that turns out to be widely missing.

Meta does have it's own density map implementation but their approach isn't compatible with our rendering engine.